tests/Unit/Core/Form/IdentifiableObject/CommandBuilder/Product/BasicInformationCommandsBuilderTest.php

```diff
@@ -58,7 +58,7 @@ public function getExpectedCommands()
         $command = new UpdateProductBasicInformationCommand($this->getProductId()->getValue());
         yield [
             [
-                'basic' => [
+                'description' => [
```

Comment:
Even command name is `BasicInformation` tho you decide to use `description`? :))

Reply:
`BasicInformation` was mostly named after the tab name back then, as I explained in another comment the tab is now called description, hence the renaming of the form field, and maybe the command should also be renamed.